modules/understanding-upgrade-channels.adoc

```diff
@@ -117,30 +94,29 @@ The presence of an update recommendation in the `stable-4` channel at any point
 endif::openshift-origin[]
 
 ifndef::openshift-origin[]
-[discrete]
-== Fast and stable channel use and strategies
+
+= Fast and stable channel use and strategies
 
 The `fast-{product-version}` and `stable-{product-version}` channels present a choice between receiving general availability releases as soon as they are available or allowing Red Hat to control the rollout of those updates. If issues are detected during rollout or at a later time, upgrades to that version might be blocked in both the `fast-{product-version}` and `stable-{product-version}` channels, and a new version might be introduced that becomes the new preferred upgrade target.
 
 Customers can improve this process by configuring pre-production systems on the `fast-{product-version}` channel, configuring production systems on the `stable-{product-version}` channel, and participating in the Red Hat connected customer program. Red Hat uses this program to observe the impact of updates on your specific hardware and software configurations. Future releases might improve or alter the pace at which updates move from the `fast-{product-version}` to the `stable-{product-version}` channel.
 endif::openshift-origin[]
 
-[discrete]
-== Restricted network clusters
+= Restricted network clusters
 
 If you manage the container images for your {product-title} clusters yourself, you must consult the Red Hat errata that is associated with product releases and note any comments that impact upgrades. During upgrade, the user interface might warn you about switching between these versions, so you must ensure that you selected an appropriate version before you bypass those warnings.
 
 ifndef::openshift-origin[]
-[discrete]
-== Switching between channels
+
+= Switching between channels
 
 A channel can be switched from the web console or through the `adm upgrade channel` command:
 
 ----
 $ oc adm upgrade channel clusterversion version --type json -p '[{"op": "add", "path": "/spec/channel", "value": "<channel>”}]'
```

Comment:
The command gets messed. Prior to 4.9, we can use `oc patch clusterversion version --type json -p '[{"op": "add", "path": "/spec/channel", "value": "<channel>”}]'` to change the channel. With 4.9 and later, we can use `oc adm upgrade channel <channel>` to change the channel

Reply:
@LalatenduMohanty @wking - Do we want to make this update with this PR, or should we create a new PR for these specific changes? If we make the update with this PR, it will cause merge conflicts. However, if we do a new PR for `4.9` then make any other updates for earlier versions, we can mitigate some of that. Since this PR is primarily focused on the reorg.